dace/frontend/fortran/gen_serde.py

```diff
@@ -0,0 +1,1368 @@
+# Copyright 2019-2025 ETH Zurich and the DaCe authors. All rights reserved.
+
```

Comment:
Please add some high-level doc comment here

Reply:
Removed the whole file.